airflow/www/forms.py

```diff
@@ -138,3 +138,12 @@ class ConnectionForm(DynamicForm):
     extra__google_cloud_platform__scope = StringField(
         lazy_gettext('Scopes (comma separated)'),
         widget=BS3TextFieldWidget())
+    extra__grpc__auth_type = StringField(
+        lazy_gettext('gRPC Auth Type'),
+        widget=BS3TextFieldWidget())
+    extra__grpc__credential_pem_file = StringField(
+        lazy_gettext('Credential pem file Path'),
```

Comment:
nit, for consistency please use camel case for each word

Reply:
please also add these extra fields for www_rbac/forms.py

Reply:
for consistency, could you rename this to `Keyfile Path`

Reply:
I checked it seems to be no more www_rbac

Reply:
ah, my bad. I'm a bit behind on the latest updates. looks like it's been moved to the default UI:)